packages/app/components/Questions/Shared/SharedComponents.tsx

```diff
@@ -35,15 +35,15 @@ export const Photo = styled(Avatar)`
 export const VerticalDivider = styled.div`
   @media (min-width: 650px) {
     border-right: 1px solid #cfd6de;
-    margin: 0 16px;
+    margin: 0 1rem;
   }
   @media (min-width: 1000px) {
-    margin: 0 32px;
+    margin: 0 2rem;
   }
 `
 export const QueueInfoColumnButtonStyle = styled(Button)<{
-  hasdemos?: boolean
-  isstudent?: boolean
+  hasdemos?: string
+  isstudent?: string
```

Comment:
Variable names sound like boolean instead of string?

Reply:
that's correct, though if i make them booleans, it gives me react warnings in the console that they can't be booleans n stuff.